http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -1104,19 +1238,19 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
     }
   }
 
-  private[util] def maybeCreateWsmIdentityAndDatabase(app: App,
-                                                      workspaceId: WorkspaceId,
-                                                      landingZoneResources: LandingZoneResources,
-                                                      namespace: KubernetesNamespace
+  private[util] def maybeCreateWsmIdentityAndDatabases(app: App,
+                                                       workspaceId: WorkspaceId,
+                                                       landingZoneResources: LandingZoneResources,
+                                                       namespace: KubernetesNamespace
   )(implicit
     ev: Ask[F, AppContext]
-  ): F[(Option[ServiceAccountName], Option[String])] = {
-    val databaseConfigEnabled = app.appType match {
-      case AppType.Wds => config.wdsAppConfig.databaseEnabled
-      case _           => false
+  ): F[(Option[ServiceAccountName], Option[Map[String, String]])] = {
```

Comment:
The function now returns a map from database name prefix to database name, rather than a single database name, since workflows app creates two databases on deployment.

Reply:
If you feel inclined you could make this a little safer (and maybe easier to use?) by having an ADT something like 

```scala
sealed trait SharedDatabaseNames
case class WdsDatabaseName(String wdsDbName) extends SharedDatabaseNames
case class WorkflowsDatabaseNames(String cbasDbName, String cromwellDbName) extends SharedDatabaseNames
```

and choosing which type to return as appropriate (might be overkill though 😄 )

Reply:
Hmm... it looks like this might [already exist](https://github.com/DataBiosphere/leonardo/pull/3603/files#diff-3d6a73abda025b49d1c9b3169828843be606585cfaeb46b798e2a567bbe1fc47R1533)?

Reply:
Yep, I actually convert it into that type when passing into the chart creation. Main reason I didn't was because WDS only has one database and I wasn't sure how to dynamically do this assignment. I think I have a marginally better understanding of Scala now than I did at the start of last week though, so I may give this another shot.